grpc/nodejs/BUILD

```diff
@@ -71,24 +71,22 @@ genrule(
     #    Output typescript to (you guessed it) the genrule output directory
     #    Set the .proto file relative path to the WORKSPACE this command is being run from
     #    Use the .proto files found in the :proto-raw-buffers filegroup as inputs.
```

Comment:
Comment needs updating if the functionality is changing

Reply:
@Phillammon addressed in https://github.com/graknlabs/protocol/pull/93/commits/97ce438beac7193c10fd7248ef76820ed1b9a59f